bridge/cypress/support/pageobjects/SequencesPage.ts

```diff
@@ -75,4 +75,9 @@ export class SequencesPage {
       .should(exists ? 'exist' : 'not.exist');
     return this;
   }
+
+  public assertServiceName(name: string, tag?: string) {
+    const serviceName = tag ? `${name}:${tag}` : name;
+    cy.get('.service-name').should('have.text', serviceName);
```

Comment:
This could possibly lead to errors in test execution, as it may not be the only element with class `.service-name` on the page. The approach we usually use is to have a `uitestid` and get the needed element by that (e.g. `uitestid="keptn-sequence-view-filter"`). All the uitestids start with `keptn-`.
We also build a cypress command `byTestId` to conveniently query the element.

Reply:
Ahh that makes sense. Here are my changes: https://github.com/keptn/keptn/pull/7283/commits/ac916a2dd370c0331f07d128b9f7781994cf056d. Thanks!